wordpress-mobile/WordPress-Android
# Reader Detail Updates (Post Header - Extract UiState)

Parent PR: #13042

This PR 

- extracts blog section from `reader_cardview_post.xml`
- extracts 
   - blog `uiState` from `ReaderPostUiState`
   - follow `uiState` from `ReaderTagHeaderViewUiState`

so that they can be reused on reader post header view.

**Merge instructions**

1. Make sure #13060 is merged
2. Change branch to "issue/12900-rip3-detail-updates-main"
3. Remove "Not Ready for Merge"
4. Merge this PR

**To test**

- Go to Reader tab
- Verify that blog section on a post card looks and behaves the same as before

PR submission checklist:
